### ISSUE TITLE
ChangeSmartArt & SmartArtColorStyle package issues

### ISSUE THREAD

**Issue by nhojpatrick** [aspose-slides/Aspose.Slides-for-Java#7]
Two classes have the wrong package declarations.

They are defined as;
package com.aspose.slides.examples.SmartArt.managingsmartartstyle;

But are in the structure for;
package com.aspose.slides.examples.SmartArt.ManageSmartArtStyle;

**Comment by saqib-razzaq-aspose** [comment]
Fixed, please refresh again from GitHub.

**Comment by nhojpatrick** [comment]
Any reason why you picked camel case names for package names.

Just wondering as both Sun and now Oracle recommend package names are all lower case, so avoid potential conflict with classes and interfaces.

https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html

**Comment by saqib-razzaq-aspose** [comment]
Thanks for asking. Yes, we know that lower case scheme is recommended for package names. But, we did this for our plugins for various platforms e.g. see https://plugins.jetbrains.com/plugin/7461?pr=idea. In one of the screenshots, you will notice there are categories like "Working with BarCode Image". The word detection in the plugins is dependent on our CamelCase package naming. If we use all lower case, the plugin would just show "workingwithbarcodeimage" which would be bad for UI and also a bad category name.

We also noticed at http://docs.oracle.com/javase/8/docs/api/ and there are many packages at the end, which use camel case scheme.
